### PR TITLE
Update home page

### PR DESCRIPTION
**Pull request by judymou**

### CODE REVIEW

File: spaceobjects/templates/spaceobjects/index.html [hunk 1]
```diff
@@ -29,13 +30,32 @@
...
 
 window.VIZ_SIMULATION_OPTS = {
   jdPerSecond: 2,
 };
 window.VIZ_OBJECT_OPTS = {};
+
+function selectOrbit(domElement) {
+  Object.keys(window.spaceobjects).forEach(function(key) {
+    const ob = window.spaceobjects[key];
+    const ob_orbit = ob.getOrbit();
```
Comment (typpo) [inline spaceobjects/templates/spaceobjects/index.html]: `obOrbit`

File: static/css/main.css [hunk 2]
```diff
@@ -213,7 +213,7 @@ ul.keyfacts li {
 }
 
 .vis-container__home {
-  height: 300px;
+  height: 300px;;
```
Comment (typpo) [inline static/css/main.css]: Remove extra `;`

File: static/css/main.css [hunk 3]
```diff
@@ -290,3 +291,29 @@ ul.keyfacts li {
...
+  display: flex;
+  flex-wrap: nowrap;
+  overflow: auto;
+  -webkit-overflow-scrolling: touch;
+  -ms-overflow-style: -ms-autohiding-scrollbar;
+}
+
+.carousel-flex-inner {
+  display: flex;
+  flex-wrap: nowrap;;
```
Comment (typpo) [inline static/css/main.css]: Remove extra `;`

File: spaceobjects/templates/spaceobjects/index.html [hunk 4]
```diff
@@ -74,36 +94,47 @@ <h4 class="subheading">Compiled data &amp; simulations for {{object_count|intcom
...
-              <li><a href="/asteroid/704-interamnia-1910-ku">704 Interamnia</a></li>
-      <!--
-            {% for space_object in space_objects %}
-              <li><a href="/asteroid/{{space_object.slug}}">{{space_object.name}}</a></li>
+          <div class="carousel-flex-container">
+            <div class="carousel-flex-inner">
+            {% for object in object_set.data %}
+              <div class="tile carousel-item" data-slug="{{object.slug}}" onmouseover="if (typeof selectOrbit !== 'undefined') selectOrbit(this)"
+                onmouseout="if (typeof deselectOrbit !== 'undefined') deselectOrbit(this)">
+                <a href="/asteroid/{{object.slug}}">
```
Comment (typpo) [inline spaceobjects/templates/spaceobjects/index.html]: Replace the href with `object.get_absolute_url`

File: spaceobjects/templates/spaceobjects/index.html [hunk 5]
```diff
@@ -74,36 +94,47 @@ <h4 class="subheading">Compiled data &amp; simulations for {{object_count|intcom
...
+                  <div>
+                    <h5>{{object.name}}</h5>
+                    <div class="tile-content">
+                      <span class="label label-danger">{{object.get_size_adjective}}</span>
```
Comment (typpo) [inline spaceobjects/templates/spaceobjects/index.html]: This is now just `object.size_adjective`

File: spaceobjects/templates/spaceobjects/index.html (hunk 1 above)
Comment (judymou) [reply spaceobjects/templates/spaceobjects/index.html]: done

File: static/css/main.css (hunk 2 above)
Comment (judymou) [reply static/css/main.css]: done

File: static/css/main.css (hunk 3 above)
Comment (judymou) [reply static/css/main.css]: done

File: spaceobjects/templates/spaceobjects/index.html (hunk 5 above)
Comment (judymou) [reply spaceobjects/templates/spaceobjects/index.html]: done

**Pull request merged**